app/build.gradle

```diff
@@ -2,7 +2,6 @@ apply plugin: 'com.android.application'
 apply plugin: 'kotlin-android'
 apply plugin: 'findbugs'
 apply plugin: 'jacoco'
-apply plugin: 'jacoco-android'
```

Comment:
Nit: Regarding this commit message: 
```
This plugin doesn't support Kotlin so we're going to configure it by
hand.
```

It might be good to drop a comment into the section that does this configuration explaining this.  Otherwise this will look like a good place to clean things up by replacing the custom code with using the plugin.

Reply:
Good idea, adding:
```kotlin
// Set up jacoco for code coverage. We originally used the jacoco-android plugin
// but it doesn't support kotlin so we're configuring it manually.